binding/dotnet/LeopardTest/MainTest.cs

```diff
@@ -49,7 +68,8 @@ private List<short> GetPcmFromFile(string audioFilePath, int expectedSampleRate)
         [ClassInitialize]
         public static void ClassInitialize(TestContext _)
         {
-            ACCESS_KEY = Environment.GetEnvironmentVariable("ACCESS_KEY");
+            //ACCESS_KEY = Environment.GetEnvironmentVariable("ACCESS_KEY");
+            ACCESS_KEY = "Nj93iN5VjmLvoeefKxrfqMXtzxYTAYiWo/tFb8JftF7IrNkpyGoyWw==";
```

Comment:
```suggestion
            ACCESS_KEY = Environment.GetEnvironmentVariable("ACCESS_KEY");
```
:wink:

Reply:
ops! good catch! I remembered I put my AccessKey somewhere to test, but I was thinking it was Java and double checked that binding my mistake!